### PR TITLE
Suggested small changes

### PR DESCRIPTION
**Pull request by wxtim**
I had a few thoughts about this - mostly small changes.

### CODE REVIEW

File: docs/whats-new-in-c8/whats-new-in-c8.md [hunk 1]
```diff
@@ -25,18 +25,17 @@ algorithm, security, UIs, and working practices, and more.
...
 - [What's Still Missing From Cylc 8](#whats-still-missing-from-cylc-8)
 - [Cylc 7 Scheduling Deficiencies Fixed by Cylc 8
 ](#cylc-7-scheduling-deficiencies-fixed-by-cylc-8)
-- [](#)
 
 ## Backward Compatibility
 [TOP](#whats-new-in-cylc-8)
 
 To make the transition easier, Cylc 8 can run Cylc 7 workflows out of the box.
+
```
Comment (wxtim) [inline docs/whats-new-in-c8/whats-new-in-c8.md:38]: I think that the line below has more impact on it's own line.

File: docs/whats-new-in-c8/whats-new-in-c8.md [hunk 2]
```diff
@@ -68,10 +67,17 @@ Cylc has been re-architected to support a remote web UI:
   - command integration, for interacting with task, jobs, and schedulers
 - a new terminal UI (TUI) as part of the CLI
 
-![Cylc Hub](./img/hub.png)
-![Cylc UI dash](./img/cylc-ui-dash.png)
-![Cylc UI tree](./img/cylc-ui-tree.png)
-![Cylc TUI](./img/cylc-tui.png)
+![title Cylc Hub](./img/hub.png)
```
Comment (wxtim) [inline docs/whats-new-in-c8/whats-new-in-c8.md:70]: I think that a bit more label, and something for text readers to get hold of might be handy.

File: docs/whats-new-in-c8/whats-new-in-c8.md [hunk 3]
```diff
@@ -132,18 +138,21 @@ Cylc 7 was aware of individual job hosts.
...
+Cylc 8 is aware of sets of host settings, specified as *platforms*
+in global configuration. By definition platform hosts share a file
+system and job runner: If one host is unavailable Cylc 8 can use
+other hosts on the same platform to interact with task jobs.
 
 ```ini
 [runtime]
    [[model]]
        platform = hpc1  # Cylc 8
+   [[model_cleanup]]
```
Comment (wxtim) [inline docs/whats-new-in-c8/whats-new-in-c8.md:153]: I can just see someone asking how they will run background jobs on hpc1...

File: docs/whats-new-in-c8/whats-new-in-c8.md [hunk 4]
```diff
@@ -191,22 +200,22 @@ workflow files into the run directory at start-up
 INSTALLED democ8 from /home/oliverh/cylc-src/democ8 -> /home/oliverh/cylc-run/democ8/run1
 
 (venv) $ cylc play democ8/run1
-                ._.                                                     
-                | |                                                     
-    ._____._. ._| |_____.                                               
-    | .___| | | | | .___|       The Cylc Workflow Engine [8.0b0]    
-    | !___| !_! | | !___.           Copyright (C) 2008-2021 NIWA        
+                ._.
```
Comment (wxtim) [inline docs/whats-new-in-c8/whats-new-in-c8.md:203]: Stupid auto format from VSCode.

File: docs/whats-new-in-c8/whats-new-in-c8.md [hunk 5]
```diff
@@ -53,7 +52,7 @@ To make the transition easier, Cylc 8 can run Cylc 7 workflows out of the box.
 ## Architecture
 [TOP](#whats-new-in-cylc-8)
 
-Cylc has been re-architected to support a remote web UI:
+Cylc has been re-architectured to support a remote web UI:
```
Comment (hjoliver) [inline docs/whats-new-in-c8/whats-new-in-c8.md:55]: "Architected" is actually the intended word (and is an actual word!).  I'll merge your PR and revert this.

shot

**Review by hjoliver — APPROVED** [review]
Thanks @wxtim :+1:

File: docs/whats-new-in-c8/whats-new-in-c8.md (hunk 5 above)
Comment (wxtim) [reply docs/whats-new-in-c8/whats-new-in-c8.md:55]: Sorry, I looked it up and missed that one.

**Pull request merged**